package.json

```diff
@@ -43,7 +43,7 @@
     },
     "devDependencies": {
         "@exabyte-io/ade.js": "2022.8.31-0",
-        "@exabyte-io/code.js": "2022.8.31-0",
+        "@exabyte-io/code.js": "git+https://github.com/Exabyte-io/code.js.git#714254037d330e32d1589a4d76791b920a2114bb",
```

Comment:
What is this commit? There's no branch in code.js for this SOF task

Reply:
The hash refers to this `code.js` branch: https://github.com/Exabyte-io/code.js/pull/34
I will update it with the latest commit hash